core/code/sidebar.js

```diff
@@ -0,0 +1,264 @@
+      }
+    }
+    return data;
+  }
+  var extra = iitc.script && iitc.script.version.match(/^\d+\.\d+\.\d+(\..+)$/);
+  extra = extra && extra[1];
+  function formatVerInfo (p) {
+    if (p.version && extra) {
+      var cutPos = p.version.length-extra.length;
+      // cut extra version component (timestamp) if it is equal to main script's one
+      if (p.version.substring(cutPos) === extra) {
+        p.version = p.version.substring(0,cutPos);
+      }
+    }
+    p.version = p.version || p.date;
+    if (p.version) {
+      var tooltip = [];
+      if (p.build) { tooltip.push('[' + p.build + ']'); }
+      if (p.date && p.date !== p.version) { tooltip.push(p.date); }
+      return L.Util.template(' - <code{title}>{version}</code>', {
+        title: tooltip[0] ? ' title="' + tooltip.join(' ') + '"' : '',
+        version: p.version
+      });
+    }
+  }
+  var plugins = pluginsInfo.map(prepData)
+    .sort(function (a,b) { return a.name > b.name ? 1 : -1; })
+    .map(function (p) {
+      p.style = '';
+      p.description = p.description || '';
+      if (p.error) {
+        p.style += 'text-decoration:line-through;';
+        p.description = p.error;
+      } else if (p.build === iitc.buildName && p.date === iitc.dateTimeVersion) { // is standard plugin
+        p.style += 'color:darkgray;';
+      }
+      p.verinfo = formatVerInfo(p) || '';
+      return L.Util.template('<li style="{style}" title="{description}">{name}{verinfo}</li>', p);
+    })
+    .join('\n');
+
+  var html = ''
+  + '<div><b>About IITC</b></div> '
+  + '<div>Ingress Intel Total Conversion</div> '
+  + '<hr>'
+  + '<div>'
+  + '  <a href="'+'@url_homepage@'+'" target="_blank">IITC Homepage</a> |' +
+    '  <a href="'+'@url_tg@'+'" target="_blank">Telegram channel</a><br />'
+  + '   On the script�s homepage you can:'
+  + '   <ul>'
+  + '     <li>Find Updates</li>'
+  + '     <li>Get Plugins</li>'
+  + '     <li>Report Bugs</li>'
+  + '     <li>Contribute!</li>'
+  + '   </ul>'
+  + '</div>'
+  + '<hr>'
+  + '<div>Version: ' + iitcVersion + '</div>';
+
+  if (window.isAndroid && android.getVersionName) {
+    html += '<div>IITC Mobile ' + android.getVersionName() + '</div>';
+  }
+  if (plugins) {
+    html += '<div><p>Plugins:</p><ul>' + plugins + '</ul></div>';
+  }
+
+  dialog({
+    title: 'IITC ' + iitcVersion,
+    id: 'iitc-about',
+    html: html,
+    width: 'auto',
+    dialogClass: 'ui-dialog-aboutIITC'
+  });
+}
+
+function setPermaLink () {
+  this.href = window.makePermalink(null, true);
+}
+
+function setupAddons () {
+  $('<a>')
+    .html('Permalink')
+    .attr({
+      id: 'permalink',
+      title: 'URL link to this map view'
+    })
+    .on({
+      mouseover: setPermaLink,
+      click: setPermaLink
+    })
+    .appendTo('#toolbox');
+
+  $('<a>')
+    .html('About IITC')
+    .attr('id', 'about-iitc')
+    .css('cursor', 'help')
+    .click(aboutIITC)
+    .appendTo('#toolbox');
+
+  window.artifact.setup();
```

Comment:
while this add a button in the toolbox, I'd rather put it back into the main boot sequence, as it's a whole intel feature

same could be said about regionscoreboard
that makes me wonder if a some `~IITC.sidebar.addToolboxLink` (to be renamed) would be a better API

Reply:
It's all about intel features, imho. Compare with `missions`.
So.. I'm not sure